proposals/collection-expression-arguments.md

```diff
@@ -398,13 +398,56 @@ Should arguments with `dynamic` type be allowed? That might require using the ru
 
 ## Open questions
 
-### Target types where arguments are *required*
+### Conversions
+
+Should collection arguments and the applicable methods affect convertibility of the collection expression?
+
+```csharp
+Print([with(comparer: null), 1, 2, 3]); // ambiguous or Print<int>(HashSet<int>)?
+
+static void Print<T>(List<T> list) { ... }
+static void Print<T>(HashSet<T> set) { ... }
+```
+
+### Type inference
+
+Should collection arguments affect type inference of the collection expression?
+
+For example, consider the following type where the constructor is called directly:
+```csharp
+UseMyCollection([with(default)]); // error: cannot infer T
+UseMyCollection([with(1)]);       // ok: UseMyCollection<int>()?
+
+static void UseMyCollection<T>(MyCollection<T> c) { ... }
+
+class MyCollection<T> : IEnumerable<T>
+{
+    public MyCollection(T arg = default) { ... }
+    public void Add(T t) { ... }
+    // ...
+}
+```
```

Comment:
similar to above, the following is not legal today: `UseMyCollection(new(1));`.  So i would expect the same to be true with CEs